duckduckgo/macos-browser
# Update Xcode to 14.3.1

Task/Issue URL: https://app.asana.com/0/1199230911884351/1204857106550931/f
Tech Design URL:
CC: @ayoy 

**Description**:

This PR updates our Xcode version to 14.3.1. This version is used by the CI, so we will need this PR in order to deploy builds from with this version of Xcode. This is important as 14.3.1 has added some Swift Concurrency fixes for macOS versions 12.0 up to (but not including) 12.1.

**Steps to test this PR**:
1. Check that the CI passes on this PR
2. Build a release build from this branch and validate that it works
3. Run the built app binary and make sure it looks good with some smoke testing. Ideally, test this on macOS 12.0, but this can admittedly be tricky to do without a VM



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
